pkg/cmd/workspace/delete.go

```diff
@@ -18,11 +19,36 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var allFlag bool
```

Comment:
@guptadeepak8, these variables should not be declared in each file but only in one. 

```
daytona ➜ /workspaces/daytona (36ee0c7) $ daytona stop --help
# github.com/daytonaio/daytona/pkg/cmd/workspace
pkg/cmd/workspace/start.go:19:5: allFlag redeclared in this block
        pkg/cmd/workspace/delete.go:22:5: other declaration of allFlag
pkg/cmd/workspace/stop.go:21:5: allFlag redeclared in this block
        pkg/cmd/workspace/delete.go:22:5: other declaration of allFlag
pkg/cmd/workspace/delete.go:9:2: "os" imported and not used
pkg/cmd/workspace/delete.go:39:26: undefined: util.ConfirmationPrompt
pkg/cmd/workspace/delete.go:122:1: missing return
```

When running your code, this is the error that occurs because you redeclared the same variable multiple times.

I suggest you declare the `allFlag` only in start.go and use it in the rest of the files. As for the `yesFlag`, that can stay in `delete.go`